business_requirement_deliverable/models/business.py

```diff
@@ -49,6 +60,40 @@ class BusinessRequirementResource(models.Model):
         string='Business Requirement Deliverable',
         ondelete='cascade'
     )
+    business_requirement_id = fields.Many2one(
+        comodel_name='business.requirement',
+        string='Business Requirement',
+        required=True,
+    )
+    business_requirement_partner_id = fields.Many2one(
+        comodel_name='res.partner',
+        related='business_requirement_id.partner_id',
+        string='Business Requirement',
+        store=True
+    )
+    business_requirement_project_id = fields.Many2one(
+        comodel_name='project.project',
+        related='business_requirement_id.project_id',
+        string='Business Requirement',
+        store=True
+    )
+
+    @api.model
+    def fields_view_get(self, view_id=None,
```

Comment:
@sudhir-serpentcs Sorry but my comment for the unnecessary explanation. 

I want to reformulate my question this it's done #103 
What is exactly is fixed by overwrite the function?